zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterManagedProcess.java

```diff
@@ -86,7 +87,7 @@ public RemoteInterpreterManagedProcess(
 
   @Override
   public String getHost() {
-    return "localhost";
+    return host;
```

Comment:
this could return null then (the initial value)?

Reply:
Right, getHost is used by the client of RemoteInterpreterManagedProcess for querying where this process is launched. And I think in semantic perspective it is right to return null when the process has not been launched.